sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexSetValue.java

```diff
@@ -196,4 +200,24 @@ public List<ApexValue<?>> getValues() {
     public Optional<String> getDefiningType() {
         return Optional.empty();
     }
+
+    // TODO: consolidate logic with ApexListValue
```

Comment:
There's some duplication here. Hoping to address it the next time I work in this area.